packages/react-vapor/src/components/modal/UnsavedChangeModalProvider.tsx

```diff
@@ -50,8 +51,8 @@ export const UnsavedChangesModalProvider: React.FunctionComponent<IUnsavedChange
             <ModalComposite
                 title={modalTitle}
                 classes={className}
-                modalHeaderClasses={['mod-confirmation']}
-                modalBodyClasses={['mod-header-padding', 'mod-form-top-bottom-padding']}
+                modalHeaderClasses={['mod-warning']}
+                modalBodyClasses={['momod-criticaldding', 'mod-form-top-bottom-padding']}
```

Comment:
What the 🦀   is that m8?

Reply:
skuze moé j'étais dans sécheuse quand j'ai écrit ça